tracer/src/Datadog.Tracer.Native/debugger_method_rewriter.cpp

```diff
@@ -1054,37 +1120,32 @@ HRESULT DebuggerMethodRewriter::ApplyMethodSpanProbe(
         rewriterWrapper.LoadLocal(returnValueIndex);
     }
 
-    // Changes all returns to a LEAVE.S (including branches to `ret`)
+    // Changes all returns to a LEAVE.S
```

Comment:
I understand that your changes no longer affect branch instructions, but does this cause any changes that the caller would observe? Or is this a change for optimization?

Reply:
This change is only for the debugger code and it fixed an issue I found while working on the Fault-Tolerant by the Exploration Testings. It threw `InvalidProgramException` (that the Fault-Tolerant was able to detect and self-heal). It's mainly a fix.

Reply:
Makes sense, thanks!